airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/WriteConfig.java

```diff
@@ -22,21 +22,24 @@
  * SOFTWARE.
  */
 
-package io.airbyte.integrations.destination;
+package io.airbyte.integrations.destination.jdbc;
 
 import io.airbyte.protocol.models.DestinationSyncMode;
 
+/**
+ * Write configuration POJO for all destinations extending {@link AbstractJdbcDestination}.
+ */
 public class WriteConfig {
 
   private final String streamName;
-  private final String outputNamespaceName;
+  private final String outputSchemaName;
```

Comment:
Since namespace is something that doesn't appear in the destination concept space now, I felt it was clearer to rename this field.